mycli/sqlcompleter.py

```diff
@@ -57,7 +59,8 @@ def __init__(self, smart_completion=True):
         self.name_pattern = compile("^[_a-z][_a-z0-9\$]*$")
 
         self.special_commands = []
-        self.table_formats = table_formats()
+        formatter = OutputFormatter()
+        self.table_formats = formatter.supported_formats()
```

Comment:
The fact that we are instantiating an `OutputFormatter` object just to get the list of supported formats is a bit of a code smell. 

It forces our `SqlCompleter` class to know about the `OutputFormatter` class. This leads to tight coupling which makes refactoring harder to do in the future. Instead, if we supply the list of supported formats as an argument to the `SqlCompleter`'s constructor then the completer object gets exactly what it needs without any coupling.